tests/core/process.c

```diff
@@ -39,6 +40,12 @@ typedef struct _TestForEachContext {
   guint number_of_calls;
 } TestForEachContext;
 
+typedef struct _TestRangeContext {
```

Comment:
Typedef and struct are usually separate, but I see that we violate this style in this file, so this is not your fault (just leave this as it is).